notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb

```diff
@@ -76,12 +76,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.                    seed_input = gr.Number(precision=0, label="Seed")
</pre><p>seed can not be lower zero, I think it make sense to add minimum value</p><p>better to use slider for steps instead of just value (min=0, max=100)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>